website/src/documentation/programming-guide.md

```diff
@@ -39,6 +39,9 @@ how to implement Beam concepts in your pipelines.
   </ul>
 </nav>
 
+{:.language-py}
+New versions of the Python SDK will only support Python 3.5 or higher. Currently, the Python SDK still supports Python 2.7.x. We recommend using the latest Python 3 version.
```

Comment:
How about the following wording:

New releases of the Python SDK will soon require Python 3.5 or higher. We recommend that new users start using Beam with Python 3.

Or:

Currently, the Python SDK still supports Python 2.7.x, however new releases of the Python SDK will soon require Python 3.5 or higher. We recommend that new users start using Beam with Python 3.

Reply:
Is there a particular release I can mention here? Or is "soon" as specific as we can get right now?

Reply:
Do you recommend this only for new users? It seems that all users would benefit from switching to Python 3

Reply:
I think we can't say more specific than soon at this point.

sure, we can drop "new" - and recommend the switch to all users.